experimental/single_number_formatter/src/provider.rs

```diff
@@ -69,9 +69,11 @@ pub struct CurrencyPatterns {
 
     /// The index of the short pattern place holder in the place holders list.
     /// If the value is u16::MAX, this means that the short pattern does not have a place holder.
+    /// If the value is u16::MAX - 1, this means that the short pattern equals to the iso code.
```

Comment:
maybe define constants like `USE_ISO_CODE = u16::MAX -1`, and whatever `u16::MAX` means

Reply:
I will use the first suggestion and we can make the other suggestion in a following PR.